actions/invite_actions.jsx

```diff
@@ -9,7 +9,8 @@ import {joinChannel} from 'mattermost-redux/actions/channels';
 import {addUsersToTeam} from 'actions/team_actions';
 
 import {t} from 'utils/i18n';
-import {isGuest, localizeMessage} from 'utils/utils';
+import {localizeMessage} from 'utils/utils';
+import { isGuest } from 'mattermost-redux/utils/user_utils';
```

Comment:
Could you remove the spaces around these imports? That's likely why the lint check on CircleCI is failing